apps/web/app/future/workflows/[workflow]/page.tsx

```diff
@@ -29,19 +25,11 @@ export const generateMetadata = async ({
   );
 };
 
-async function getProps(context: GetServerSidePropsContext) {
-  const safeParams = querySchema.safeParse(context.params);
-
-  console.log("Built workflow page:", safeParams);
-  if (!safeParams.success) {
-    return notFound();
-  }
-  return { workflow: safeParams.data.workflow };
-}
+const getData = withAppDirSsg(getStaticProps);
 
 export const generateStaticParams = () => [];
 
-export default WithLayout({ getLayout: null, getData: getProps, Page: LegacyPage })<"P">;
+export default WithLayout({ getLayout: null, getData, Page: LegacyPage })<"P">;
```

Comment:
If we dont use WithLayout in the `layout.tsx` files, we can simplify WithLayout HOC

Reply:
this further optimization can go in a separate PR